dask/array/tests/test_array_core.py

```diff
@@ -357,6 +372,7 @@ def test_field_access():
     assert eq(y[['b', 'a']], x[['b', 'a']])
 
 
+@pytest.mark.skipif(int(np.__version__[2]) < 8, reason="Need numpy > 1.8")
```

Comment:
Same here, it would be unfortunate not to test any reductions in numpy 1.7 just because one or two of the functions is missing.